testing/00_title.md

```diff
@@ -44,162 +73,235 @@ plugins: RevealMarkdown, RevealChalkboard, RevealHighlight, RevealMath.KaTeX, Re
 ## Test Types
 
 <ul>
-  <li>Unit test
-  <ul class="fragment fade-up" data-fragment-index="1">
-    <li>Smallest possible unit</li>
+  <li class="fragment fade-up">Exceptions in the code base
+  <ul>
+    <li>Intended to handle "expected" problems</li>
+    <li>Sound an alarm as soon as the problem arises</li>
+    <li>Provide clear feedback to the user</li>
+  </ul></li>
+  <li class="fragment fade-up">Unit testing
```

Comment:
Maybe highlight Unit testing to make it clear that we only will focus on them? Perhaps put them on the top of the list.

Reply:
Good point. I will find a way to emphasize that this is what we will focus on. I still like the order the way it is (I organized it more or less small-large scale/simple-complex, which I think makes sense), but will think of some other way to emphasize.